app/assets/scss/app.scss

```diff
@@ -268,3 +268,39 @@ $govuk-new-link-styles: true;
       height: 60px;
     }
   }
+
+
+// Overide prism.js code colours
+.token.punctuation{
+  color: govuk-colour("dark-blue") !important;
+}
```

Comment:
Pri idsk komponentoch sme sa snazili vyhnut " !important " ale mozno pri code snippete to nieje az taky problem kedze toto nebudeme pouzivat v govcms ani vo wp